ui/src/app/certificate_requests/row.tsx

```diff
@@ -67,7 +67,14 @@ export default function Row({ id, csr, certificate, ActionMenuExpanded, setActio
             <td id="expanded-row" className="p-table__expanding-panel" aria-hidden={detailsMenuOpen ? "false" : "true"}>
                 <div className="row">
                     <div className="col-8">
-                        <p><b>Common Name</b>: {csrObj.subjects.find((e) => e.type == "Common Name")?.value}</p>
+                        <p><b>Common Name</b>: {csrObj.commonName || "N/A"}</p>
+                        <p><b>Organization</b>: {csrObj.organization || "N/A"}</p>
+                        <p><b>Country</b>: {csrObj.country || "N/A"}</p>
+                        <p><b>Locality</b>: {csrObj.locality || "N/A"}</p>
+                        <p><b>Email Address</b>: {csrObj.emailAddress || "N/A"}</p>
+                        <p><b>Subject Alternative Name DNS</b>: {csrObj.sansDns && csrObj.sansDns.length > 0 ? csrObj.sansDns.join(', ') : "N/A"}</p>
+                        <p><b>Subject Alternative Name IP addresses</b>: {csrObj.sansIp && csrObj.sansIp.length > 0 ? csrObj.sansIp.join(', ') : "N/A"}</p>
+                        <p><b>Certificate request for a certificate authority</b>: {csrObj.is_ca ? "Yes" : "No"}</p>
```

Comment:
Currently there is a lot of `N/A`'s showing up cluttering vertical space and it's not obvious to the eye where to look. Here are a couple of ideas: 

1. Have the key (whatever the cert field is) and Value (`N/A`) be grey instead of black for cases where the value is empty. This would make it clearer where to look.
2. Don't show the cert key/value at all when they are empty. This would reduce the number of unnecessary vertical space.

My recommended approach would be to go with option 2. In either case I would recommend experimenting with these 2 and any other idea you may have. 

![image](https://github.com/canonical/gocert/assets/18486508/56dc4ad2-e67f-4cd2-a6cc-50dce10faab0)

Reply:
It would also be nice to reduce the margins between these fields to have it be a bit more compact.

Reply:
I like seeing the N/A values because I usually forget what fields a CSR or a Cert could have, and it could be useful for users to quickly identify if they forgot to fill out a field if they don't have the fields memorized. Since i'm not the target audience for this, im open to having it either way.

Reply:
I pushed some changes greying out missing info and reducing the margins.
Here is how it looks now.
![Uploading Screenshot 2024-06-26 at 12.43.55.png…]()

Reply:
I experimented with not displaying the missing info. And yes it would like nicer.
But I believe having the N/A values for missing fields provides good useful information as it would tell the user that those values were not provided, which is not necessary a trivial thing to get from not showing the fields.
Here is how it would look.
![Screenshot 2024-06-26 at 11 36 12](https://github.com/canonical/gocert/assets/26838825/c99716f3-c53c-44e5-bcc5-ffc1eb6cef6c)

Reply:
I still think it's valuable information. But I don't have a better argument for now, so I changed it.
Done.